presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -86,6 +86,11 @@ int32_t SystemConfig::numSpillThreads() const {
   return opt.hasValue() ? opt.value() : std::thread::hardware_concurrency();
 }
 
+std::string SystemConfig::spillerSpillPath() const {
+  auto opt = optionalProperty<std::string>(std::string(kSpillerSpillPath));
+  return opt.hasValue() ? opt.value() : "";
```

Comment:
Should we have this "" string stored as a default like other defaults?

Reply:
If we don't always need a spill path, perhaps we need to check the spill path being configured under some condition? Otherwise, we might need a default spill path.

Reply:
> Should we have this "" string stored as a default like other defaults?

@tanjialiang 
I thought about it and with it being actually empty, which means "disabled spilling", I don't see much point in that.

Reply:
> If we don't always need a spill path, perhaps we need to check the spill path being configured under some condition? Otherwise, we might need a default spill path.

@gggrace14 
Empty path in Presto means spilling is disabled, so no default.